src/commands/check.yml

```diff
@@ -17,6 +17,12 @@ parameters:
       Specify the root path containing the shell scripts. The directory will be recursively searched.
     type: string
     default: "."
+  ignore_dirs:
+    description: |
+      Specifiy directories to be ignored during shellcheck. Each directory provided should be quoted.
+      The directory will be recursively searched.
```

Comment:
This description is a touch confusing on first glance - I _think_ you mean it's space delineated, but not sure. Maybe we can clean this up a bit.

Reply:
What I was trying to express here is that the directories need to be quoted. How they're separated doesn't actually matter. However, after trying to verify I realized that I might not have tested this on dirs with spaces. Going to fix this and then try and see if I can make the verbiage any clearer.